src/components/common/TokenInputField/TokenInputField.module.scss

```diff
@@ -0,0 +1,36 @@
+.api-token-input {
+    width: 100%;
+    height: 40px;
+    padding: 0 12px;
+    font-size: 14px;
+    line-height: 1.43;
+    border: none;
+    border-radius: 4px;
+    color: #c2c2c2;
+    background-color: #0e0e0e;
+    -webkit-border-radius: 4px;
+    -moz-border-radius: 4px;
+    -ms-border-radius: 4px;
+    -o-border-radius: 4px;
+}
+
+.btn-authenticate {
+    cursor: pointer;
+    width: 118px;
+    height: 42px;
+    padding: 10px 16px;
+    border: none;
+    border-radius: 0 4px 4px 0;
+    background-color: #ff444f;
+    font-size: 14px;
+    line-height: 1.43;
+    color: #fff;
+
+    &:hover {
+        background-color: #d43e47;
+      }
```

Comment:
please run option-shift-F in this file to prettify tabs 😊

Reply:
as u wish, my mistress